recipes/msdfgen/all/conanfile.py

```diff
@@ -49,11 +49,11 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("freetype/2.12.1")
+        self.requires("freetype/2.13.2")
         if  Version(self.version) < "1.10":
             self.requires("lodepng/cci.20200615")
         else:
-            self.requires("libpng/1.6.39")
+            self.requires("libpng/1.6.40")
```

Comment:
```suggestion
            self.requires("libpng/[>=1.6 <2]")
```
version range is now allowed for libpng